sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -298,6 +303,7 @@ private[ui] class ExecutionPagedTable(
 
     val headerRow: Seq[Node] = {
       executionHeadersAndCssClasses.map { case (header, sortable) =>
+        val headerTooltip = headerNameWithTooltips.getOrElse(header,"")
```

Comment:
Hi, @07ARB . Could you run `dev/scalastyle` and fix this line?
As you see, `GitHub Action` failed due to this.

Reply:
Ok